crates/stark_hash/build.rs

```diff
@@ -1,14 +1,11 @@
-use crate::curve::*;
+use std::env;
 use std::fs;
+use std::path::Path;
 
-#[allow(dead_code)]
-pub fn generate_consts(path: &str, bits: u32) {
-    let buf = &mut String::with_capacity(10 * 1024 * 1024);
+use stark_curve::*;
 
-    buf.push_str("//! Autogenerated, don't modify.\n");
-    buf.push_str("//! See [`crate::curve_consts_gen::generate_consts`].\n");
-    buf.push_str("//! The constants are used by [`crate::hash::pedersen_hash_preprocessed`].\n\n");
-    buf.push_str("use crate::curve::AffinePoint;\n\n");
+fn generate_consts(path: &Path, bits: u32) {
```

Comment:
```suggestion
fn generate_consts(path: &Path, bits: std::num::NonZeroU8) {
```

This would probably be the cheapest way to write down that 0 is not a good selection.